### PR TITLE
Move the strategies to tilings

### PR DESCRIPTION
**Pull request by enadeau**
Closes #83 

- [x] Verification strategies
- [x] Inferral strategies
- [x] Decomposition strategies
- [x] Equivalence strategies
- [x] Batch strategies
- [ ] Move the tests for the strategies' function from `tilescope` to `tilings`

### CODE REVIEW

**Comment by enadeau** [comment]
-Is there any situation where we want to ignore parent of a requirement insertion strategy? I've ported it but I'm not sure we need it

File: tilings/strategies/equivalence.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    """
+    Strategy that place a single forced point of a point requirement.
+
+    The point placement strategy considers all point requirements in their own
+    requirement lists. For each of them, it returns a new tiling where the
+    point has been placed with a force.
+    """
+    yield from partial_requirement_placement(tiling, point_only=True)
+
+
```
Comment (christianbean) [inline tilings/strategies/equivalence.py]: Fusion strategies should have a seperate file as they are not really "equivalence" strategies.

File: tilings/strategies/batch.py [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+def partial_row_and_col_placements(tiling, **kwargs):
+    req_placements = (RequirementPlacement(tiling, own_row=False),
+                      RequirementPlacement(tiling, own_col=False))
+    for req_placement in req_placements:
+        yield from req_placement.all_row_placement_rules()
+        yield from req_placement.all_col_placement_rules()
+
+
+# TODO: Clarify what is this function doing
+def requirement_list_placement(tiling, **kwargs):
```
Comment (enadeau) [inline tilings/strategies/batch.py]: I do not really understand what this strategy is doing? Can we clarify the docstring?

File: tilings/strategies/equivalence.py (hunk 1 above)
Comment (enadeau) [reply tilings/strategies/equivalence.py]: They were all in equivalence strategies in tilescope but I also prefer having its own file. I'll change that.

Comment (enadeau) [reply tilings/strategies/equivalence.py]: Done.

File: tilings/strategies/batch.py (hunk 2 above)
Comment (enadeau) [reply tilings/strategies/batch.py]: We decided to remove it

File: tilings/strategies/equivalence.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+def all_placements(tiling, **kwargs):
+    req_placements = (RequirementPlacement(tiling),
+                      RequirementPlacement(tiling, own_row=False),
+                      RequirementPlacement(tiling, own_col=False))
+    for req_placement in req_placements:
+        yield from req_placement.all_point_placement_rules()
+        yield from req_placement.all_requirement_placement_rules()
+        yield from req_placement.all_col_placement_rules()
+        yield from req_placement.all_row_placement_rules()
+
```
Comment (christianbean) [inline tilings/strategies/equivalence.py]: Should all placements be in batch ? it has both batch and equivalence.

Comment (enadeau) [reply tilings/strategies/equivalence.py]: Henning and I agreed with that. I changed it.

**Pull request merged**